gravitee-apim-console-webui/src/management/api/documentation-v4/documentation-edit-page/api-documentation-v4-edit-page.component.html

```diff
@@ -26,6 +26,7 @@ <h3>{{ pageTitle }}</h3>
       </div>
     </div>
     <div class="header__actions">
+      <button mat-stroked-button *ngIf="mode === 'edit'" (click)="deletePage()">Delete page</button>
```

Comment:
Maybe add a check here that the delete button is disabled if the current page is used as `generalConditions`?